addons/point_of_sale/views/pos_payment_method_views.xml

```diff
@@ -9,6 +9,7 @@
                     <widget name="web_ribbon" title="Archived" bg_color="text-bg-danger" invisible="active"/>
                     <field name="active" invisible="1"/>
                     <field name="type" invisible="1"/>
+                    <field name="default_pos_receivable_account_name" invisible="1" />
```

Comment:
Can you try removing this here and see if the placeholder still works? I think you don't need to declare a field in a view as invisible to use it.

Reply:
Its not working,
I think we have to declare it in a view, in order to use it in `options`.